caas/kubernetes/clientconfig/k8s_test.go

```diff
@@ -243,70 +241,16 @@ func (s *k8sConfigSuite) TestGetSingleConfig(c *gc.C) {
 	})
 }
 
-func (s *k8sConfigSuite) TestConfigErrors(c *gc.C) {
```

Comment:
Do we still need these tests? I guess we're using the k8 SDK config loader

Reply:
Yep